pnpm-lock.yaml

```diff
@@ -6867,7 +6878,7 @@ packages:
     peerDependencies:
       tailwindcss: '>=2.0.0 || >=3.0.0 || >=3.0.0-alpha.1'
     dependencies:
-      tailwindcss: 3.2.4_ts-node@10.9.1
+      tailwindcss: 3.2.4_v776zzvn44o7tpgzieipaairwm
```

Comment:
Just wondering why the tailwindcss dependencies changed in the lockfile, maybe we could revert anything not related to @aws-sdk/types in this lockfile?

Reply:
Yeah it's a bit weird - I'll go through and revert any that don't make sense